docker/Dockerfile

```diff
@@ -19,6 +27,7 @@ RUN sh /harden.sh && rm /harden.sh
 
 # Copy source code and install packages
 COPY --from=builder /install /usr/local
+COPY --from=go_builder /go/cosign/main /app/cosign/cosign
```

Comment:
We must be aware that we change the official docker image to include an experimental binary. Maybe we should release a version with and without cosign as long as cosign is experimental/we support this feature experimentally, what do you think?

Reply:
thought about it and felt like it will cause significant extra work. If you do not choose to use it, it will never be called. So I feel it is safe enough :shrug:

Reply:
It's not only about security though, image size also goes up from 56,5 to 82,4 MB, which is quite significant for at least currently being an experiment of an experiment. What about having an experimental Dockerfile (for connaisseur:$VERSION-experimental), which in this case would include cosign and e.g. replacing the cosign binary with the `false` binary in the main image?

Reply:
With the base image being 44,9MB, the cosign update is double the size of the Connaisseur footprint. In the past, we've had the issue of A's Internet connection not being able to download the image in time for a two-minute sentinel timeout (and also had tickets regarding that) and the increase for an experimental inclusion of an experimental signing method would increase this size by 46%. I see your reasoning with feature flags, but I feel like the size increase is unwarranted given the very experimental nature